runtime/autoload/health/provider.vim

```diff
@@ -392,18 +392,6 @@ function! s:check_python(version) abort
     let python_exe = ''
   endif
 
-  " Check if $VIRTUAL_ENV is valid.
-  if exists('$VIRTUAL_ENV') && !empty(python_exe)
-    if $VIRTUAL_ENV ==# matchstr(python_exe, '^\V'.$VIRTUAL_ENV)
-      call health#report_info('$VIRTUAL_ENV matches executable')
-    else
-      call health#report_warn(
-        \ '$VIRTUAL_ENV exists but appears to be inactive. '
-        \ . 'This could lead to unexpected results.',
-        \ [ 'If you are using Zsh, see: http://vi.stackexchange.com/a/7654' ])
```

Comment:
why remove the hint about zshell? this was a painful discovery.

Reply:
The link was relocated below, as you've probably noticed since then? I also didn't think it quite appropriate to explicitly mention Zsh in the given context, as I don't believe it's a Zsh-specific issue -- any shell can have rc files which override the PATH set by the virtualenv and inherited by Nvim.